recipes/pydusa/meta.yaml

```diff
@@ -3,8 +3,8 @@ package:
     version: 1.15
 
 source:
-    url: https://github.com/cryoem/pydusa/archive/master.tar.gz
-    fn: pydusa-master.tar.gz
+    url: https://github.com/cryoem/pydusa/archive/{{GIT_PYDUSA_BRANCH}}.tar.gz
+    fn: pydusa-{{GIT_PYDUSA_BRANCH}}.tar.gz
```

Comment:
Same as the previous line.

`{{ environ.get(GIT_PYDUSA_BRANCH) }}`

Reply:
According to the documenatation 

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#templating-with-jinja

One only needs environ if one want to set a default value if the variable is empty.

Reply:
In my installation test, the variable is also inherited by conda build if the variable is exported.

Reply:
OK, then! Learned something new.